### PR TITLE
fix: [PLG-3039]: UI changes for CD Get Started and Wizard Page

### PR DESCRIPTION
**Pull request by devinder-harness**
### Summary

This PR adds:

Fullscreen mode to CD Get Started and CD Onboarding Wizard
Updates github clone cmd
Fixes issues with GitOps Agent Identifier

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/GitopsDeploymentSetup.tsx [hunk 1]
```diff
@@ -89,11 +97,12 @@ export default function GitopsDeploymentSetup({
             allowCopy
             ignoreWhiteSpaces={false}
             commandSnippet={getCommandStrWithNewline([
-              getString(
+              `${getString(
                 'cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.pipelineSetupStep.commands.clonecmd',
                 { gitUser: 'harness-community' }
-              ),
-              getString('cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.pipelineSetupStep.commands.cddir')
+              )} && ${getString(
```
Comment (abhinavrastogi-harness) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/GitopsDeploymentSetup.tsx:103]: why do we have `&&` operator with strings on both sides?

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/CDOnboardingFullScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  const onStateChange = (): void => {
+    setScreenType(OnbaordingScreenType.WIZARD)
+  }
+  return (
+    <Dialog
+      onClose={gotoPipelines}
+      isOpen={true}
+      style={{
```
Comment (abhinavrastogi-harness) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/CDOnboardingFullScreen.tsx:45]: can we avoid inline styles?

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/GitopsDeploymentSetup.tsx (hunk 1 above)
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/GitopsDeploymentSetup.tsx:103]: thats the requirement 
image

**Review by abhinavrastogi-harness — DISMISSED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
LGTM

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**